frontend/src/components/pages/potions/PotionListItem.js

```diff
@@ -0,0 +1,57 @@
+import Image from 'next/image';
+
+import { faCross, faStar, faPerson, faPlus, faEdit } from '@fortawesome/free-solid-svg-icons';
+
+import { Card, CardActions, CardContent, CardHeader, CardMedia, Grid, Typography } from '@mui/material';
+
+import { getPotionDifficultyColor } from '../../../lib/utils';
+import Link from '../../Link';
+import ListItemBox from '../../ui/ListItemBox';
+
+const PotionListItem = ({ potion }) => {
+	const { name, slug, effect, side_effects, characteristics, difficulty, inventors, image } = potion.attributes;
+
+	return (
+		<Grid item xs={12} sm={6} md={4} lg={3}>
+			<Card
+				sx={{
+					height: '100%',
+					display: 'flex',
+					flexDirection: 'column',
+					border: `3px solid ${getPotionDifficultyColor(difficulty)}`,
+					justifyContent: 'space-between',
+				}}
+			>
+				<CardMedia>
+					<Image
+						as='image'
+						src={image || '/images/missing_image.jpg'}
+						alt={`Picture of ${name}`}
+						width='100%'
+						height='100%'
+						layout='responsive'
+						objectFit='scale-down'
+						quality={30}
+						loading='eager'
+						priority
+					/>
+				</CardMedia>
+				<CardHeader title={name} />
+				<CardContent>
+					<ListItemBox value={difficulty || 'Unknown'} icon={faStar} />
+					<ListItemBox value={inventors || 'Unknown'} icon={faPerson} />
+					<ListItemBox value={side_effects || 'Unknown'} icon={faCross} />
+					<ListItemBox value={effect || 'Unknown'} icon={faEdit} />
+					<ListItemBox value={characteristics || 'Unknown'} icon={faPlus} />
```

Comment:
Can you remove the 'Unknown' from all of the ListItemBoxes? It should only render those if there's a value for each of them, which is done in the ListItemBox itself, so no need to add the 'Unknown'.